nerd/client/nerdapi.go

```diff
@@ -81,38 +80,45 @@ func (nerdapi *NerdAPIClient) url(p string) string {
 	return url.String()
 }
 
-func (nerdapi *NerdAPIClient) handleError(err error, aerr *payload.Error, resp *http.Response) error {
+func (nerdapi *NerdAPIClient) doRequest(s *sling.Sling, successV interface{}) *APIError {
+	e := &payload.Error{}
+	req, err := s.Request()
+	if err != nil {
+		//TODO: should error message include more details like URL, HTTP method and payload (sling is not very verbose in giving detailed error information)?
```

Comment:
True. So we probably want Sling to be more verbose on what kind of error occurred. So I still think it's a good idea to add some proper error handling to Sling, instead of including it at this point in the code base.